package.json

```diff
@@ -24,6 +24,6 @@
     "bootstrap": "./bootstrap"
   },
   "dependencies": {
-    "geonames.js": "2.3.0"
```

Comment:
Did you remove `geonames.js` dependency?

Reply:
Since I'm not using fetch, we don't need this. I tested and this works.